src/main/resources/fabric-test-api-v1.mixins.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I don't think this and `src/main/resources/assets/fabric-test-api-v1/icon.png` should be there.

Reply:
Good spot, not sure how those got there.